### PR TITLE
Vulnerabilities and recommendations redesign

### PR DESCRIPTION
**Pull request by chasulin**
First batch of below-the-fold-redesign items ([mocks](https://www.figma.com/file/JGqemgZwlG9e5DQbZpdKgy/Redesign?node-id=1370%3A88195)):
- Recommendations
- Vulnerabilities (+ 'About this data' modal)
- Removes region name from below each section's header
- Share buttons moved from above to below each section (and removes compare 'More info' modal)

### CODE REVIEW

File: src/components/Compare/CompareTable.tsx [hunk 1]
```diff
@@ -295,7 +273,21 @@ const CompareTable = (props: {
               </TextButton>
             )}
           </div>
-          <MoreInfoButton onClick={onClickFAQ} />
+          <ShareImageButtons
```
Comment (chasulin) [inline src/components/Compare/CompareTable.tsx:276]: moves share buttons under compare (in place of the 'More Info' button that triggered the now unused FAQ modal)

File: src/components/LocationPage/ChartBlock.tsx [hunk 2]
```diff
@@ -55,35 +55,25 @@ function ChartBlock(props: {
...
         <>
           <MetricChart
             metric={metric}
             projections={projections}
             height={chartHeight}
           />
-          <DisclaimerWrapper>{disclaimerContent}</DisclaimerWrapper>
+          <LocationPageSectionFooter>
+            <DisclaimerWrapper>{disclaimerContent}</DisclaimerWrapper>
+            <ShareButtons
```
Comment (chasulin) [inline src/components/LocationPage/ChartBlock.tsx:71]: moves share buttons under chart

File: src/components/LocationPage/ShareButtons.style.ts [hunk 3]
```diff
@@ -1,31 +1,13 @@
...
 
-const WrapperStyles = css`
-  display: block;
+export const Wrapper = styled.div`
   position: relative;
-  height: 0;
-  z-index: 1;
-`;
-
-export const DesktopButtonsWrapper = styled.div`
```
Comment (chasulin) [inline src/components/LocationPage/ShareButtons.style.ts:14]: now that the share buttons render in the same place on both mobile+desktop, we dont need these wrappers

File: src/components/LocationPage/ShareButtons.style.ts [hunk 4]
```diff
@@ -37,51 +19,25 @@ export const ButtonContainerStyles = css`
   width: fit-content;
 `;
 
-export const SaveOrShareContainer = styled.div`
-  ${ButtonContainerStyles};
-  height: auto;
-  margin-left: auto;
-`;
-
-export const SaveOrShareButton = styled.div<{
```
Comment (chasulin) [inline src/components/LocationPage/ShareButtons.style.ts:46]: also an unnecessary wrapper

File: src/components/LocationPage/LocationPageBlock.style.ts [hunk 5]
```diff
@@ -5,24 +5,24 @@ import {
 } from 'assets/theme/sizes';
 
 export const BlockContainer = styled.div`
-  margin: 3rem 1rem 0;
+  margin: 0 1rem 4.5rem;
```
Comment (chasulin) [inline src/components/LocationPage/LocationPageBlock.style.ts:8]: when unifying the location page spacing, i spaced each section via the top margin. this moves the spacing to the bottom margin instead (and increases it a little bit in accordance with the location page mocks)

File: src/components/LocationPage/ShareButtons.style.ts [hunk 6]
```diff
@@ -92,18 +46,17 @@ export const SocialShareButton = styled.div<{ isLast?: boolean }>`
   }
 `;
 
-export const CopyLinkButton = styled.div`
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  font-size: 0.775rem;
-  line-height: 1.3;
+export const CopyLinkButton = styled(MuiButton).attrs(props => ({
```
Comment (chasulin) [inline src/components/LocationPage/ShareButtons.style.ts:49]: this button was previously just a div. changed to a mui button

File: src/components/LocationPage/ShareButtons.tsx [hunk 7]
```diff
@@ -1,33 +1,30 @@
...
-} from './ShareButtons.style';
+import { Wrapper } from './ShareButtons.style';
 import ShareButton from 'components/NewLocationPage/ShareButton/ShareButton';
 import { ClickAwayListener } from '@material-ui/core';
 import makeChartShareQuote from 'common/utils/makeChartShareQuote';
 import * as urls from 'common/urls';
 import { Region } from 'common/regions';
 
-interface InnerContentProps {
+const ShareButtons: React.FC<{
```
Comment (chasulin) [inline src/components/LocationPage/ShareButtons.tsx:11]: this file had 2 components: a main (`ShareButtons`) and an inner component (`InnerContent`). did a bunch of cleanup, deleted some unnecessary wrappers, and combined the components into a single one (`ShareButtons`)

File: src/components/LocationPageSectionFooter/LocationPageSectionFooter.tsx [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Wrapper } from './LocationPageSectionFooter.style';
+
+const LocationPageSectionFooter: React.FC = ({ children }) => {
```
Comment (chasulin) [inline src/components/LocationPageSectionFooter/LocationPageSectionFooter.tsx:4]: i have bigger plans for this shared footer, but punting to a later pr, so for now its just a single wrapping div 🙃

File: src/components/Explore/Explore.style.ts [hunk 9]
```diff
@@ -15,9 +15,7 @@ function palette(props: any) {
   return props.theme.palette.chart;
 }
 
-export const Container = styled.div`
-  padding-bottom: 1.5rem; // Leaves some space below the 'xx days ago' tooltip
-`;
+export const Container = styled.div``;
```
Comment (pnavarrc) [inline src/components/Explore/Explore.style.ts:18]: Maybe this can be removed?

**Review by pnavarrc — APPROVED** [review]
Works great! discussed over slack: We need to enable analytics tracking for this component and maybe tweak the event label to make it more generic

**Comment by chasulin** [comment]
@pnavarrc [updated the tracking and made the label more generic](https://github.com/covid-projections/covid-projections/pull/3576/commits/d703e297a02498d19b665b77ada9aa75bfacf46f) 👍🏼

**Pull request merged**